Extensions/InternetConnectivity.json

```diff
@@ -0,0 +1,32 @@
+{
+  "author": "The Gem Dev",
+  "description": "",
+  "extensionNamespace": "",
+  "fullName": "Internet Connectivity",
+  "name": "InternetConnectivity",
+  "shortDescription": "an extension which checks if the device is connected to the internet. ",
```

Comment:
Change short description to: `A condition to check if the device is connected to the internet.`